content/announcements/2024-12-10-contribfest-kubecon-eu-2025.md

```diff
@@ -0,0 +1,52 @@
+
+At this year's cert-manager ContribFest, maintainers including CyberArk's Maël Valais and Richard Wall, and Zenior's Erik Godding Boye will be on hand to show users of all experience levels how to get involved with the cert-manager project.
+
+## Good First Issues
+
+- **[#6637](https://github.com/cert-manager/cert-manager/issues/6637)**: Enable a lint to help avoid accidental shadowing of package imports.
+- **[#6695](https://github.com/cert-manager/cert-manager/issues/6695)**: Update the Helm chart’s READMEs to mention usage in K3d clusters.
+- **[#6622](https://github.com/cert-manager/cert-manager/issues/6622)**: Improve logging clarity in ACME challenge solver.
+- **[#6732](https://github.com/cert-manager/cert-manager/issues/6732)**: Add a missing comment or doc string for a function in the E2E tests.
+- **[#5925](https://github.com/cert-manager/cert-manager/issues/5925)**: Use `readOnlyRootFilesystem: true` for all containers.
+- **[#5514](https://github.com/cert-manager/cert-manager/issues/5514)**: Venafi Issuer to read `caBundle` from ConfigMap or Secret.
+- **[#2943](https://github.com/cert-manager/cert-manager/issues/2943)**: Allow configuration of the keystore file name.
+- **[#1686](https://github.com/cert-manager/cert-manager/issues/1686)**: Support for Microsoft Active Directory Certificate Services.
+- **[#2744](https://github.com/cert-manager/cert-manager/issues/2744)**: Documenting resolution for DigitalOcean and HTTP01 "connection timed out" error.
+- **[#2888](https://github.com/cert-manager/cert-manager/issues/2888)**: Allow configuration of `MaxConcurrentChallenges`.
+- **[#1520](https://github.com/cert-manager/cert-manager/issues/1520)**: Ingress-shim: Skip updating Certificate resource if one already exists.
+
+## How to Prepare
+
+To make the most of the event, here are a few tips:
+
+- Familiarize yourself with the cert-manager project by exploring our [GitHub repository](https://github.com/cert-manager/cert-manager).
+- Read our [Contributor Guide](https://cert-manager.io/docs/contributing/) to understand the basics of contributing to cert-manager.
+- Join our community on the [Kubernetes Slack](https://slack.k8s.io/) in the `#cert-manager-dev` channel to ask questions and connect with the team.
+- Don't forget your laptop!
+
+## Join Us!
+
+If you're in London, you can join us for ContribFest on Wednesday, April 2nd, **(details TBD)**.
+
+
+We’re looking forward to seeing you at KubeCon EU 2025 in London! Together, we’ll make cert-manager even better while fostering a stronger open-source community.
+
+**See you at ContribFest!**
+
+Stay connected by following us on [Twitter](https://twitter.com/cert_manager), joining the channels `#cert-manager` and `#cert-manager-dev` on the [Kubernetes Slack](https://slack.k8s.io/), and exploring our [Contributor Guide](https://cert-manager.io/docs/contributing/).
```

Comment:
suggestion: Twitter handle looks incorrect, and I'd vote we should link to our slack guide directly rather than repeating it here.

```suggestion
Stay connected by following us on [BlueSky](https://bsky.app/profile/cert-manager.bsky.social), [Mastodon](https://infosec.exchange/@CertManager), [Twitter](https://twitter.com/certmanager), joining us [on Slack](https://cert-manager.io/docs/contributing/#slack), and exploring our [Contributor Guide](https://cert-manager.io/docs/contributing/).
```